src/main/kotlin/no/nav/amt/arena/acl/goldengate/ArenaMessageProcessor.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.amt.arena.acl.goldengate
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import no.nav.amt.arena.acl.domain.ArenaData
+import no.nav.amt.arena.acl.domain.arena.ArenaWrapper
+import no.nav.amt.arena.acl.repositories.ArenaDataIdTranslationRepository
+import no.nav.amt.arena.acl.repositories.ArenaDataRepository
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class ArenaMessageProcessor(
+	private val arenaDataRepository: ArenaDataRepository,
+	private val arenaDataIdTranslationRepository: ArenaDataIdTranslationRepository
+) {
+
+	private val log = LoggerFactory.getLogger(this::class.java)
+
+	private val mapper = jacksonObjectMapper()
+
+	fun handleArenaGoldenGateRecord(record: ConsumerRecord<String, String>) {
+		val data = mapper.readValue(record.value(), ArenaWrapper::class.java).toArenaData()
+
+		if (isArenaDataIrrelevant(data)) {
+			log.info("Arena data is not relevant and will be ignored."
+				+ " topic=${record.topic()} partition=${record.partition()} offset=${record.offset()}"
+				+ " table=${data.arenaTableName} arenaId=${data.arenaId}"
+			)
+
+			return
+		}
+
+		arenaDataRepository.upsert(data)
+	}
+
+	private fun isArenaDataIrrelevant(arenaData: ArenaData): Boolean {
+		val arenaDataTranslation = arenaDataIdTranslationRepository.get(arenaData.arenaTableName, arenaData.arenaId)
```

Comment:
For at de skal være ignored=true så må de allerede vært forsøkt ingested så det er vel bare endringer på gjennomføringer/deltakere vi allerede kjenner til..

Reply:
Jeg vet ikke helt hvordan det er tenkt, om vi skal lagre unna deltakere i translation tabellen som vi ikke er interessert i eller om vi skal kun lagre tiltaksgjennomføringer som vi ikke er interessert i slik at vi vet om vi trenger å lagre deltakere når de dukker opp